apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -534,7 +533,7 @@ func (m *ModelManagerAPI) CreateModel(ctx context.Context, args params.ModelCrea
 	// mode and don't make the calls so test can keep passing.
 	// THIS IS VERY TEMPORARY.
```

Comment:
Should this be a todo? How temporary is it?

Reply:
Temporary while we move logic around. Already discussed in the PR it was introduced and it lives in the risk register.